public/images/edit_button.svg

```diff
@@ -0,0 +1,48 @@
+<svg width="80" height="80" viewBox="0 0 80 80" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<title>edit_button</title>
+<desc>Created using Figma</desc>
```

Comment:
Do we really need `Created using Figma` here?

Reply:
lol, no. All the svg/design programs add something like that by default. I will remove it.